### PR TITLE
[REF] inventory: restructure shipping section

### PR DESCRIPTION
**Pull request by Felicious**
Restructure docs in the **Shipping and Receiving** section, according to @cru-odoo's suggestions

## Summary of changes
```
🟢 ADD
❌ REMOVE
🔘 RENAME/MOVE

inventory_and_mrp/
└── inventory/
    ├── product_management/
    │   ├── 🟢 configure.rst
    │   ├── 🟢 configure/
    │   ├── ❌ product_replenishment.rst --> configure.rst
    │   ├── ❌ product_replenishment/
    │   │   ├── 🔘 mto.rst --> ../../warehouse_storage/replenishment/mto.rst
    │   │   ├── 🔘 reordering_rules.rst ../../warehouse_storage/replenishment/reordering_rules.rst
    │   │   ├── 🔘 strategies.rst --> ../../warehouse_storage/replenishment.rst
    │   │   └── 🔘 uom.rst --> ../configure/uom.rst
    │   └── product_tracking/
    │       ├── 🔘 package.rst --> ../configure/package.rst
    │       ├── 🔘 packaging.rst --> ../configure/packaging.rst
    │       └── ❌ differences.rst --> ../product_tracking.rst
    ├── shipping_receiving/
    │   ├── 🟢 picking_methods.rst
    │   ├── 🟢 picking_methods/
    │   ├── setup_configuration/
    │   │   └── ❌ delivery_method.rst --> ../setup_configuration.rst
    │   ├── advanced_operations_shipping/
    │   │   ├── 🔘 dropshipping.rst --> ../daily_operations/dropshipping.rst
    │   │   └── 🔘 scheduled_dates.rst --> ../../warehouses_storage/replenishment/lead_times.rst
    │   └── daily_operations/
    │       └── ❌ shipments_deliveries.rst --> ../daily_operations.rst
    └── warehouse_storage/
        ├── ❌ advanced_operations_warehouse.rst
        ├── ❌ advanced_operations_warehouse/
        │   ├── 🔘 cross_dock.rst --> ../../shipping_receiving/daily_operations/cross_dock.rst
        │   ├── 🔘 owned_stock.rst --> ../../shipping_receiving/daily_operations/owned_stock.rst
        │   ├── 🔘 putaway.rst --> ../../shipping_receiving/daily_operations/putaway.rst
        │   ├── 🔘 stock_warehouses.rst --> ../../shipping_receiving/daily_operations/stock_warehouses.rst
        │   ├── 🔘 storage_category.rst --> ../../shipping_receiving/daily_operations/storage_category.rst
        │   ├── 🔘 removal.rst --> ../../shipping_receiving/removal_strategies.rst
        │   ├── 🔘 batch_transfers.rst --> ../../shipping_receiving/picking_methods/batch.rst
        │   ├── 🔘 cluster_picking.rst --> ../../shipping_receiving/picking_methods/cluster.rst
        │   └── 🔘 wave_transfers.rst --> ../../shipping_receiving/picking_methods/wave.rst
        └── inventory_management/
            ├── 🔘 resupply_warehouses.rst --> ../replenishment/resupply_warehouses.rst
            ├── 🔘 warehouse_replenishment_transfer.rst --> ../replenishment/warehouse_replenishment_transfer.rst
            └── 🔘 use_routes.rst --> ../../shipping_receiving/daily_operations/use_routes.rst
```
## Fwport
yes... but I might do a separate PR since there will be some redirect rules I need to write. We'll see!!!

[Task](https://github.com/odoo/documentation/pull/10398)

Note: thank you @samueljlieber for mapping out all the files in tree format!

### CODE REVIEW

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations.rst:1]: Dear reviewer:
Please resist the urge to improve this doc for content :see_no_evil: 
It's a copy and paste of [Incoming shipments and delivery orders](https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.html), and if there's an interest in improving this doc, let's do it in a follow-up [IMP] PR :pray:

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst:1]: Dear reviewer: please resist the urge to improve this doc for content, as well :see_no_evil: 

An improvement of this doc is available in [17.0](https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.html), with each removal method detailed in its own doc. If there's an interest to back-port this content, let's do that in a follow-up [ADD] doc :blush:

**Comment by dade-odoo** [comment]
I removed the request for a localizations review since the implication to accounting docs is so minimal, but if you do really want one from us, don't hesitate to reach out!

**Comment by Felicious** [comment]
Hello @samueljlieber! The product expert approved of the change through DMs, so I'm passing this chonky Inventory restructure doc to you for review!

File: redirects/16.0.txt [hunk 1]
```diff
@@ -54,27 +54,37 @@ applications/inventory_and_mrp/manufacturing/quality_control/quality_control_poi
...
 applications/inventory_and_mrp/inventory/management/warehouses.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst                                                                       # /inventory_and_mrp/inventory/management/warehouses -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management
 applications/inventory_and_mrp/inventory/management/warehouses/resupply_warehouses.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/resupply_warehouses.rst                               # /inventory_and_mrp/inventory/management/warehouses/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
+applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/resupply_warehouses.rst applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/resupply_warehouses.rst                    # /inventory_and_mrp/inventory/warehouses_storage/inventory_management/* -> /inventory_and_mrp/inventory/warehouses_storage/replenishment/*
```
Comment (samueljlieber) [inline redirects/16.0.txt:78]: Add redirect for `storage_category.rst`
```suggestion
applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/resupply_warehouses.rst applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/resupply_warehouses.rst                    # /inventory_and_mrp/inventory/warehouses_storage/inventory_management/* -> /inventory_and_mrp/inventory/warehouses_storage/replenishment/*
applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/storage_category.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/storage_category.rst              # /inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/storage_category -> /inventory_and_mrp/inventory/shipping_receiving/daily_operations/storage_category
```

File: redirects/16.0.txt [hunk 2]
```diff
@@ -54,27 +54,37 @@ applications/inventory_and_mrp/manufacturing/quality_control/quality_control_poi
...
 applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouse_replenishment_transfer.rst     # /inventory_and_mrp/inventory/management/warehouses/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
+applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouse_replenishment_transfer.rst applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/warehouse_replenishment_transfer.rst # /inventory_and_mrp/inventory/warehouses_storage/inventory_management/* -> /inventory_and_mrp/inventory/warehouses_storage/replenishment/*
```
Comment (samueljlieber) [inline redirects/16.0.txt:80]: update comment :)
`/*` would mean the entire directory `inventory_management/` is being redirected, and that is not the case
```suggestion
applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouse_replenishment_transfer.rst applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/warehouse_replenishment_transfer.rst # /inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouse_replenishment_transfer -> /inventory_and_mrp/inventory/warehouses_storage/replenishment/warehouse_replenishment_transfer
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @Felicious! 
Awesome work on this restructure! 

I am requesting changes because only a single redirect is missing, and then a handful of the redirect comments need to be adjusted. Please see my suggestions below!

I also mapped out the changes in this PR because the description does not seem to be up-to-date/accurate to the changes. Could you please update the description too?
```
🟢 ADD
❌ REMOVE
🔘 RENAME/MOVE

inventory_and_mrp/
└── inventory/
    ├── product_management/
    │   ├── 🟢 configure.rst
    │   ├── 🟢 configure/
    │   ├── ❌ product_replenishment.rst --> configure.rst
    │   ├── ❌ product_replenishment/
    │   │   ├── 🔘 mto.rst --> ../../warehouse_storage/replenishment/mto.rst
    │   │   ├── 🔘 reordering_rules.rst ../../warehouse_storage/replenishment/reordering_rules.rst
    │   │   ├── 🔘 strategies.rst --> ../../warehouse_storage/replenishment.rst
    │   │   └── 🔘 uom.rst --> ./configure/uom.rst
    │   └── product_tracking/
    │       ├── 🔘 package.rst --> ../configure/package.rst
    │       ├── 🔘 packaging.rst --> ../configure/packaging.rst
    │       └── ❌ differences.rst --> ../product_tracking.rst
    ├── shipping_receiving/
    │   ├── 🟢 picking_methods.rst
    │   ├── 🟢 picking_methods/
    │   ├── setup_configuration/
    │   │   └── ❌ delivery_method.rst --> ../setup_configuration.rst
    │   ├── advanced_operations_shipping/
    │   │   ├── 🔘 dropshipping.rst --> ../daily_operations/dropshipping.rst
    │   │   └── 🔘 scheduled_dates.rst --> ../../warehouses_storage/replenishment/lead_times.rst
    │   └── daily_operations/
    │       └── ❌ shipments_deliveries.rst --> ../daily_operations.rst
    └── warehouse_storage/
        ├── ❌ advanced_operations_warehouse.rst
        ├── ❌ advanced_operations_warehouse/
        │   ├── 🔘 cross_dock.rst --> ../../shipping_receiving/daily_operations/cross_dock.rst
        │   ├── 🔘 owned_stock.rst --> ../../shipping_receiving/daily_operations/owned_stock.rst
        │   ├── 🔘 putaway.rst --> ../../shipping_receiving/daily_operations/putaway.rst
        │   ├── 🔘 stock_warehouses.rst --> ../../shipping_receiving/daily_operations/stock_warehouses.rst
        │   ├── 🔘 storage_category.rst --> ../../shipping_receiving/daily_operations/storage_category.rst
        │   ├── 🔘 removal.rst --> ../../shipping_receiving/removal_strategies.rst
        │   ├── 🔘 batch_transfers.rst --> ../../shipping_receiving/picking_methods/batch.rst
        │   ├── 🔘 cluster_picking.rst --> ../../shipping_receiving/picking_methods/cluster.rst
        │   └── 🔘 wave_transfers.rst --> ../../shipping_receiving/picking_methods/wave.rst
        └── inventory_management/
            ├── 🔘 resupply_warehouses.rst --> ../replenishment/resupply_warehouses.rst
            ├── 🔘 warehouse_replenishment_transfer.rst --> ../replenishment/warehouse_replenishment_transfer.rst
            └── 🔘 use_routes.rst --> ../../shipping_receiving/daily_operations/use_routes.rst
```
^ Made with [tree.nathanfriend.io](https://tree.nathanfriend.io/?s=(%27options!(%27fancy!true~fullPath!false~zilGSlash!false~rootDot!false)~source!(%27source!%27%5Cn4O_and_mrp%2FIO%2FII4HXCBe8CBV*9H6LBeZ9HR2mto3J7ERmtoYU%20JJ7ERU2sztegix3J7E6YuomL.%2FBVuomZ*HzckG%2F2Qe3BVQeYQG3BVQGZ49differencx3HzckGZ4I4jCA8CA%2F**q%2F*49delivery_method3qZ*NM%2F2dropM35dropMYscheduled_datx3J7sERlead_timxZ5*49shipments_deliverix3dailyFZ4I47E*9N7Z9N7%2F2%2205%22Yowned_W05owned_WYputaway05putawayYW_7s05W_7sYT05TZ42removal0removal_sztegixZ42batchKs0A%2FbatchYcluster_pickG0A%2FclusterYwaveKs0A%2FwaveZ44*O_X2%233R%23Y7_6K3R7_6KYuse_routx05use_routxZ*4%27)~version!%271%27)*I4403Jj2*4%F0%9F%94%98%203LJ4%20%205dailyF%2F6replenishment7warehouse8.rst9%E2%9D%8C%20ApickG_methodsBconfigurC*%F0%9F%9F%A2%20E_storagVF_operationsGingHproduct_I%5Cn44J..%2FK_znsferL8%20--%3E%20MshippGNadvancedF_OinventoryQpackagR6%2FTstorage_categoryUreorderG_rulx8Ve%2FWstockXmanagement%2FY82Z8*jM_receivG%2Fqsetup_Bationxesztra%22cross_dock%23rxupply_7s%01%23%22zxqjZYXWVUTRQONMLKJIHGFECBA987654320*)

Other than the one redirect and adjusting the comments, everything else looks good to me! 

Please tag me once more for another quick look, thank you for your work!

**Comment by Felicious** [comment]
Hi @samueljlieber ! This restructure PR is ready for another look :blush: 

Appreciate your help SO SO SOO much. Mapping out all the files and what's been deleted/renamed/moved is such a smart, clear, and succinct way to track all the changes in the files. Truly grateful for your thorough, kind, and patient review, as always :heart:

File: redirects/16.0.txt [hunk 3]
```diff
@@ -54,27 +54,38 @@ applications/inventory_and_mrp/manufacturing/quality_control/quality_control_poi
...
+applications/inventory_and_mrp/inventory/product_management/product_replenishment/uom.rst applications/inventory_and_mrp/inventory/product_management/configure/uom.rst                                                       # /inventory_and_mrp/inventory/product_management/product_replenishment/* -> /inventory_and_mrp/inventory/product_management/configure/*
+applications/inventory_and_mrp/inventory/product_management/product_tracking/package.rst applications/inventory_and_mrp/inventory/product_management/configure/package.rst                                                    # /inventory_and_mrp/inventory/product_management/product_tracking/* -> /inventory_and_mrp/inventory/product_management/configure/*
+applications/inventory_and_mrp/inventory/product_management/product_tracking/packaging.rst applications/inventory_and_mrp/inventory/product_management/configure/packaging.rst                                                # /inventory_and_mrp/inventory/product_management/product_tracking/* -> /inventory_and_mrp/inventory/product_management/configure/*
```
Comment (samueljlieber) [inline redirects/16.0.txt:65]:
```suggestion
applications/inventory_and_mrp/inventory/product_management/product_replenishment/uom.rst applications/inventory_and_mrp/inventory/product_management/configure/uom.rst                                                       # /inventory_and_mrp/inventory/product_management/product_replenishment/uom -> /inventory_and_mrp/inventory/product_management/configure/uom
applications/inventory_and_mrp/inventory/product_management/product_tracking/package.rst applications/inventory_and_mrp/inventory/product_management/configure/package.rst                                                    # /inventory_and_mrp/inventory/product_management/product_tracking/package -> /inventory_and_mrp/inventory/product_management/configure/package
applications/inventory_and_mrp/inventory/product_management/product_tracking/packaging.rst applications/inventory_and_mrp/inventory/product_management/configure/packaging.rst                                                # /inventory_and_mrp/inventory/product_management/product_tracking/packaging -> /inventory_and_mrp/inventory/product_management/configure/packaging
```

File: redirects/16.0.txt [hunk 4]
```diff
@@ -83,38 +94,50 @@ applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_
...
 applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/label_type.rst                                            # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
 applications/inventory_and_mrp/inventory/shipping/operation/dropshipping.rst applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping.rst                                        # /inventory_and_mrp/inventory/shipping/operation/* -> /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/*
+applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/dropshipping.rst                       # /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/* -> /inventory_and_mrp/inventory/shipping_receiving/daily_operations/*
```
Comment (samueljlieber) [inline redirects/16.0.txt:140]:
```suggestion
applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping.rst applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/dropshipping.rst                       # /inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/dropshipping -> /inventory_and_mrp/inventory/shipping_receiving/daily_operations/dropshipping
```

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious! 
Thank you for your kind words 😄

I have just a couple more suggestions to update the comments of some of the redirects. These werent a _directory-to-directory_ move, so `/*` doesnt apply, since not all files in the dir were moved to the same dir :)

Please implement these and then everything else looks good to me! 

We can go ahead and try for the FWP, I reviewed 17.0 and the structure looks mostly the same for the files changed in this PR, so I anticipate it going relatively smoothly. Please handle any conflicts ASAP, thank you!

Thank you for your work!
...
@robodoo delegate=Felicious

**Review by StraubCreative — APPROVED** [review]
Logging my approval here as well, which was given prior in DM as long as the suggestions I shared were considered and implemented where there was agreement. 

Good lift, Fel! 💪

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**